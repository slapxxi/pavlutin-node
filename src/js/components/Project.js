import React from 'react';
import { Container } from './Project.styled';

function Project({ project, key }) {
  return (
    <Container key={key}>
      <h1>{ project.title }</h1>
    </Container>
  );
}

export default Project;
