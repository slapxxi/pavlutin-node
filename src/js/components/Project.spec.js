import React from 'react';
import render from 'react-test-renderer';
import Project from './Project';

const project = { title: 'Test' };

it('renders project', () => {
  const tree = render.create(<Project project={project} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
