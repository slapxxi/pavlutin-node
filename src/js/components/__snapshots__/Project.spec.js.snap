// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders project 1`] = `
<li
  className="gEoxXY"
>
  <h1>
    Test
  </h1>
</li>
`;
